content/tutorials/manage-the-rippled-server/installation/capacity-planning.md

```diff
@@ -59,42 +67,40 @@ path={path_to_ledger_store}
 
 <!-- {# ***TODO***: Add link to online_delete section, when complete, per https://ripplelabs.atlassian.net/browse/DOC-1313  #} -->
 
 ### Log Level
 
-The default `rippled.cfg` file sets the logging verbosity to `warning`. This setting greatly reduces disk space and I/O requirements over more verbose logging. However, more verbose logging provides increased visibility for troubleshooting.
+The default `rippled.cfg` file sets the logging verbosity to `warning` in the `[rpc_startup]` stanza. This setting greatly reduces disk space and I/O requirements over more verbose logging. However, more verbose logging provides increased visibility for troubleshooting.
+
+**Caution:** If you omit the `log_level` command from the `[rpc_startup]` stanza, `rippled` writes logs to disk at the `debug` level and outputs `warning` level logs to the console. `debug` level logging requires several more GB of disk space per day than `warning` level, depending on transaction volumes and client activity.
+
 
-**Caution:** If you omit the `log_level` command from the `[rpc_startup]` stanza, `rippled` writes logs to disk at the `debug` level and outputs `warning` level  logs to the console. `debug` level logging requires several more GB of disk space per day than `warning` level, depending on transaction volumes and client activity.
 
 ## Network and Hardware
 
-Each `rippled` server in the XRP Ledger network performs all of the transaction processing work of the network. It is unknown when volumes will approach maximum network capacity. Therefore, the baseline hardware for production `rippled` servers should be similar to that used in Ripple's [performance testing](https://ripple.com/dev-blog/demonstrably-scalable-blockchain/).
+Each `rippled` server in the XRP Ledger network performs all of the transaction processing work of the network. ***TODO: Is this true? What do we mean by "transaction processing" -- do we mean validation? Do we mean storing transaction history? I ask because I don't think "each rippled server in the XRP Ledger network" participates in the same way. Depending on the answer - aren't there some servers on the network that don't participate in "transaction processing"?*** It is unknown when volumes will approach maximum network capacity. ***TODO: This is true - but do we need to say it?*** Therefore, the baseline hardware for production `rippled` servers should be similar to that used in Ripple's [performance testing](https://ripple.com/dev-blog/demonstrably-scalable-blockchain/). ***TODO: Not sure about the word "Therefore" here - it's saying that because we don't know when volumes will approach max network capacity - you could use this configuration. I'm not sure if that make sense. I may be misunderstanding what we mean.***
```

Comment:
> Depending on the answer - aren't there some servers on the network that don't participate in "transaction processing"?

You're definitely right that the original phrase is too broad and vague. I'll the other questions for @mDuo13 since he undoubtedly has good answers for them. For this particular question, my understanding is that you're right: not all servers "process transactions." _Every server_ is active in the network in the sense that it communicates with other servers. Only validators "process transactions," as far as I know.

Reply:
Actually, it's true that **all `rippled` servers process transactions.** Every server receives the same set of transactions, applies them individually to its closed ledger, and calculates the result. Non-validating servers don't speak up about their results, but all servers do the processing. If they get the same resulting hash as the one approved by consensus, then they know they ended up with the exact same ledger as everyone else. If not, they have to throw out their results and download the approved results, but the rest of the time they save themselves the trouble by calculating it themselves. (This also serves as a check that everyone is processing transactions legitimately.)